docs/guides/qpu-information.mdx

```diff
@@ -45,7 +45,7 @@ These values are shown on both the [Compute resources tab](https://quantum.ibm.c
 
 *   **Name** - The unique name assigned to a specific QPU. QPUs hosted on IBM Cloud® have names that begin with `ibmq_*` (older QPUs) or `ibm_*` (newer QPUs). All QPUs are given a city name, e.g., `ibmq_johannesburg`. This name does not indicate where the actual QPU is hosted. They are named after IBM&reg; locations around the world.
 *   **Qubits** - The number of qubits in a QPU. For physical QPUs, this is the number of physical qubits in the device. For simulators, this number need not be uniquely defined, and instead can depend on the simulation method and/or the amount of memory available.
-*   **EPLG** - Error per layered gate in a chain of 100 qubits. Error per layered gate measures the average gate process error in a layered chain of $N$ qubits ($N$=100 here). It is derived from a similar quantity known as the layer fidelity (LF) where EPLG$_{100}$ = 1-LF$^{\frac{1}{99}}$ and layer fidelity is the process fidelity of the layered chain of $N$ qubits. For details, see the paper [Benchmarking quantum processor performance at scale](https://arxiv.org/abs/2311.05933).
+*   **EPLG** - Average error per layered gate in a chain of 100 qubits. Average error per layered gate measures the average gate error in a layered chain of $N$ qubits ($N$=100 here). It is derived from a similar quantity known as the layer fidelity (LF) where EPLG$_{100}$ = 4/5(1-LF$^{\frac{1}{99}}$) and layer fidelity is the process fidelity of the layered chain of $N$ qubits. For details, see the paper [Benchmarking quantum processor performance at scale](https://arxiv.org/abs/2311.05933). Note that in the paper EPLG is defined for process error and here we quote average gate error for consistency with the individually reported gate errors.
```

Comment:
@dcmckayibm Sorry to be the IBM style police but we have to avoid using "we" in general, though I know it's standard in academic writing. I could rewrite with the passive voice but wonder if there's a better way - suggestions?

Reply:
changed, does this work?